Sources/MapboxCoreNavigation/NavigationService.swift

```diff
@@ -321,14 +321,6 @@ public class MapboxNavigationService: NSObject, NavigationService {
     }
     
     public func start() {
-        // Jump to the first coordinate on the route if the location source does
```

Comment:
Can you please comment why this is no longer needed?

Reply:
Hmm, I think this file is already outdated. This change was reverted back.